pkg/kotsutil/kots.go

```diff
@@ -59,6 +60,17 @@ var (
 	}
 )
 
+type OverlySimpleGVK struct {
+	APIVersion string               `yaml:"apiVersion"`
+	Kind       string               `yaml:"kind"`
+	Metadata   OverlySimpleMetadata `yaml:"metadata"`
+}
+
+type OverlySimpleMetadata struct {
+	Name      string `yaml:"name"`
+	Namespace string `yaml:"namespace"`
+}
+
```

Comment:
Are these types defined in two places now?  If it differs slightly, maybe we can use embedded structs?

Reply:
hmm, there is a quirk that's making consolidating the two types a bit annoying, so i'm gonna keep this duplicate for now since it's more of a utility type.